book/website/reproducible-research/make.md

```diff
@@ -27,7 +27,7 @@ the way to the pdf file of the paper!
 <small style="margin: 5pt auto; text-align: center; display: block;">A
 schematic for a research project that uses LaTeX.</small>
 
-## An Introduction to Make
+## Why This is Useful
```

Comment:
```suggestion
## An Introduction to Make
```

Reply:
I will rever this one, as I think in the new template we are no longer using the "why this is useful" - apologies that haven't yet merged the new template PR. That should be my next PR review :D